FireBrick/UI/AuthViewController/AuthViewController.swift

```diff
@@ -56,102 +53,110 @@ class AuthViewController: UIViewController, GIDSignInUIDelegate {
         
         Auth.auth().signIn(with: credential) { (authResult, error) in
             guard error == nil else {
-                self.showErrorMsg(errorTxt: error?.localizedDescription)
+                self.showErrorMessage(errorTxt: error?.localizedDescription)
                 return
             }
             self.userDidSignIn()
         }
     }
     
-    @objc func googleErrorAuth(_ notification:Notification) {
+    @objc
+    private func googleErrorAuth(_ notification:Notification) {
         stopGoogleLoader()
         guard let data = notification.userInfo else {
             return
         }
         guard let error = data["error"] as? NSError else {
-            showErrorMsg(errorTxt: "Uknown authentication error")
+            showErrorMessage(errorTxt: "Uknown authentication error")
             return
         }
         guard error.code != Constants.errorCodes.googleUserCancel else {
             return
         }
-        showErrorMsg(errorTxt: error.localizedDescription)
+        showErrorMessage(errorTxt: error.localizedDescription)
     }
     
-    @objc func emailButtonTap(_ sender: UITapGestureRecognizer) {
-        let storyboard = UIStoryboard(name: "EmailAuth", bundle: nil)
-        let vc = storyboard.instantiateViewController(withIdentifier: "e-mail auth"  ) as! EmailAuthViewController
+    @objc
+    private func emailButtonTap(_ sender: UITapGestureRecognizer) {
+        let storyboard = UIStoryboard(name: "EmailAuthViewController", bundle: nil)
+        guard let vc = storyboard.instantiateViewController(withIdentifier: "e-mail auth"  ) as? EmailAuthViewController else {
```

Comment:
```suggestion
        guard let vc = storyboard.instantiateViewController(withIdentifier: "EmailAuthViewController") as? EmailAuthViewController else {
```
also need to change id in storyboard